lib/mobility-core/src/Kernel/External/Call/Exotel/Config.hs

```diff
@@ -30,6 +30,7 @@ import Kernel.Prelude
 data ExotelCfg = ExotelCfg
   { exotelUrl :: BaseUrl,
     callbackUrl :: BaseUrl,
+    url :: BaseUrl, -- applet url for Call Flow (IVR, greeting etc.)
```

Comment:
What is this url? It is backward incompatible, else we may need to update it in db before rolling out.

Reply:
this url we can create by adding a call flow in exotel dashboard, this is required to play the audio for reminders, Yes we need to update it in db before rolling out